astropy/modeling/functional_models.py

```diff
@@ -282,6 +317,35 @@ def __init__(self, amplitude=amplitude.default, x_mean=x_mean.default,
             amplitude=amplitude, x_mean=x_mean, y_mean=y_mean,
             x_stddev=x_stddev, y_stddev=y_stddev, theta=theta, **kwargs)
 
+    @property
+    def bbox(self):
+        '''
+        Returns the bounding box limits (x_low, x_high, y_low, y_high)
+        Default is (-5 * x_stddev, 5 * x_stddev, -5 * y_stddev, 5 * y_stddev)
+        relative to the source position.
+
+        model.bbox = 'auto' will reset the default.
+        '''
+        if self._bbox == 'auto':
+            x_width = 5 * self.x_stddev
```

Comment:
I think you have to consider that the Gaussian can be rotated (Just like you did for `Ellipse2D`)